### DISCUSSION TITLE
<Your question or feedback about the GitHub Discussions product>

### DISCUSSION THREAD

**Discussion by mickey1995** [community/community#110438] [Discussions]
### Select Topic Area

General

### Body

I’m sure this will get deleted right away because of the embarrassment but you guys did a KYC for VC spectra I mean I have a snapshot of it on my phone and where are they guys they just seem to pack up and disappear I wrote you guys three months ago regarding it never got a response back of course actually, my ticket was deleted but I just want to know or let you know that since you KY seed, I will be getting a hold of my attorney and this will be taken care of. I’m getting my money back whether anybody likes it or not do you wanna reason why I bought this versus patch it was because you guys KYc it

**Comment by queenofcorgis** [comment]
Hi @mickey1995 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on the project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.


In our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. This type of post is considered "off topic" for our Community Discussions. Any repeat and future violations may result in a temporary or indefinite block from the Community. Thanks for understanding. 


Best of luck!
